### PR TITLE
Feat: Update doc for emea.

### PR DESCRIPTION
**Pull request by marcos-albornoz**
Closes VC-26900

### CODE REVIEW

**Review by luispresuelVenafi — APPROVED** [review]

**Review by JigarAtVenafi — APPROVED** [review]

**Review by rvelaVenafi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ TPP REST APIs but please be advised this goes against Venafi recommendations.
 
 ## Prerequisites for using with Venafi as a Service
 
-1. The Venafi as a Service REST API is accessible at https://api.venafi.cloud from the system where VCert
+1. The Venafi as a Service REST API is accessible at https://api.venafi.cloud or https://api.eu.venafi.cloud(if you have an EU account) from the system where VCert
```
Comment (sbanic-venafi) [inline README.md:69]: Resolve if you are sure we need to use `api.eu.venafi.cloud` (as opposed to `api.venafi.eu`).

**Review by sbanic-venafi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rvelaVenafi) [inline README.md:69]: missing blank space between the eu url and the parenthesis
```
https://api.eu.venafi.cloud (if you have an EU account)
```

**Review by rvelaVenafi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (marcos-albornoz) [reply README.md]: Confirmed that the `api.venafi.eu` should be used and changed to use it

Comment (marcos-albornoz) [reply README.md]: Done!

**Review by sbanic-venafi — APPROVED** [review]

**Pull request merged**